inlong-sdk/transform-sdk/src/test/java/org/apache/inlong/sdk/transform/process/TestTransformStringFunctionsProcessor.java

```diff
@@ -118,6 +119,20 @@ public void testLocateFunction() throws Exception {
         Assert.assertEquals(1, output5.size());
         Assert.assertEquals(output5.get(0), "result=null");
     }
+
+    @Test
+    public void testReplaceFunction() throws Exception {
+        String transformSql = "select replace(string1, string2, string3) from source";
+        TransformConfig config = new TransformConfig(transformSql);
+        TransformProcessor<String, String> processor = TransformProcessor
+                .create(config, SourceDecoderFactory.createCsvDecoder(csvSource),
+                        SinkEncoderFactory.createKvEncoder(kvSink));
+        // case1: replace('hooray', 'oray', 'lly')
+        List<String> output = processor.transform("hooray|oray|lly", new HashMap<>());
+        Assert.assertEquals(1, output.size());
+        Assert.assertEquals(output.get(0), "result=holly");
+    }
```

Comment:
It is better to add more test cases to cover the common usage scenarios. In this example, you can add a case that string `s` doesn't contains string `s1`.

Reply:
Thanks,i will add more case.